packages/asset-swapper/src/utils/market_operation_utils/constants.ts

```diff
@@ -462,6 +472,9 @@ export const MAINNET_TOKENS = {
     CRV: '0xd533a949740bb3306d119cc777fa900ba034cd52',
     MIM: '0x99d8a9c45b2eca8864373a26d1459e3dff1e17f3',
     EURT: '0xc581b735a1688071a1746c968e0798d642ede491',
+    // Synapse ecosystem
+    nUSD: '0x1B84765dE8B7566e4cEAF4D0fD3c5aF52D3DdE4F',
```

Comment:
Should we convert it to lowercase address?

Reply:
thank you! some of the pools were uppercase as well